src/workerd/server/server.c++

```diff
@@ -1523,10 +1523,10 @@ kj::Own<Server::Service> Server::makeWorker(kj::StringPtr name, config::Worker::
     ActorCacheSharedLruOptions getActorCacheLruOptions() override {
       // TODO(someday): Make this configurable?
       return {
-        .softLimit = 16ull << 20,
-        .hardLimit = 128ull << 20,
+        .softLimit = 16 * (1ull << 20), // 16 MB
```

Comment:
I know it shouldn't matter so feel free to ignore me, but it's making me sad to see this all as MB rather than MiB.

Reply:
![image](https://user-images.githubusercontent.com/7085343/216666268-254b7951-83c3-4db3-9342-bff0525a813d.png)

Reply:
I put `MiB` and then I was sad all our configuration options say `Mb`, so we were already lying in one place or another. I can say `MiB` here and/or change the config names, though.

Reply:
Ignore me, I was conflating the defaults here and the config mulitplier: https://bitbucket.cfdata.org/projects/ew/repos/edgeworker/browse/src/edgeworker/scheduling/limit-enforcer-impl.c++?at=b739b846301e1d62ebef0acc94548a7ba30632a9#277

Either way it hurt my head, I'll fix the comments here.

Reply:
Eh I agree that it's confusing and that it's probably more downside to updating the internal config names than it's worth. No need to do anything.